pino.js

```diff
@@ -128,7 +133,7 @@ function asJson (obj, msg, num) {
   var hasObj = obj !== undefined && obj !== null
   var objError = hasObj && obj instanceof Error
   msg = !msg && objError ? obj.message : msg || undefined
-  var data = this._baseLog + this._lscache[num] + this.time()
+  var data = '{' + this._lscache[num] + this.time()
```

Comment:
can you embed `{` into `this._lscache`?

Reply:
@mcollina not sure if this is what you wanted:

Reply:
??

Reply:
You asked: 

> can you embed { into this._lscache?

So I provided some examples above and what I thought that meant. Apologies if I misinterpreted the ask.

Reply:
added one more commit. let me know if that is what you were requesting.